src/main/drivers/bus_spi_hal.c

```diff
@@ -284,6 +277,33 @@ bool spiTransfer(SPI_TypeDef *instance, uint8_t *out, const uint8_t *in, int len
     return true;
 }
 
+static bool spiBusReadBuffer(const busDevice_t *bus, uint8_t *out, int len)
+{
+#define SPI_DEFAULT_TIMEOUT 10
```

Comment:
It needs to be provided to HAL functions - is it the right value though? should there be a SPI_NO_TIMEOUT. Its also defined here within the spiBusReadBuffer, but used outside its scope (see below). not that it matters, but for me its something I would expect at the top of the file.